### DISCUSSION TITLE
Does SE have an audio to text tool?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5639] [General]
Something to Auto generate subs so I can edit them

**Comment by niksedk** [comment]
Some SE beta versions had support for PocketSphinx speech recognition, and I also tested Mozilla Deepspeech - but I never got any of them working well with SE.

In latest beta you can try "Video" - "Video/audio to text..." : https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SubtitleEditBeta.zip

It requires clear speech without too much background noise and not too fast talking.

Let me know how it works for you - if at all.

**Comment by DrReddP** [comment]
Do you know what tiktok uses? That's what got me thinking because it's
super easy to create subs using their tools. Even YouTube auto generated
would be a place to start

**Comment by niksedk** [comment]
It's in the Video menu - Video/audio to text...

(It will use the current video loaded in SE)

![image](https://user-images.githubusercontent.com/1261331/148975282-5dce5b12-b7e7-4259-842b-1ad8a5cdab32.png)

**Reply by DrReddP** [reply]
Excellent. Thank you

**Comment by formlessdao** [comment]
Does SE  3.6.5 beta not let you import the larger 1,8 gigabyte vosk-model-en-us-0.22 ?

**Reply by niksedk** [reply]
You can manually download any models and save in the "Models" folder, which you can find here:

![image](https://user-images.githubusercontent.com/1261331/159151642-a50f4ef7-11e3-4958-8ca4-714624f3230d.png)
